wvd-templates/wvd-management-ux/deploy/mainTemplate.json

```diff
@@ -33,6 +33,20 @@
             "metadata": {
                 "description": "Provide a unique name of the web app that you wish to create"
             }
+        },
+        "_artifactsLocation": {
+            "type": "string",
+            "metadata": {
+                "description": "The base URI where artifacts required by this template are located."
+            },
+            "defaultValue": "https://raw.githubusercontent.com/Azure/RDS-Templates/master/wvd-templates/wvd-management-ux/deploy/scripts/"
```

Comment:
Please change this to "https://raw.githubusercontent.com/Azure/RDS-Templates/master/wvd-templates/wvd-management-ux/deploy/" . Then, for the scripts below, you'll need to add "scripts/" before them.